katello/katello-backup

```diff
@@ -262,39 +324,42 @@ if @dir.nil?
   puts "**** ERROR: Please specify an export directory ****"
   puts optparse
   exit(-1)
-else
-  puts "Starting backup: #{Time.now}"
-  create_directories(@dir.dup)
-  FileUtils.cd @dir
-  generate_metadata
-  FileUtils.cp Dir.glob(File.join(@options[:incremental], '.*.snar')), @dir if @options[:incremental]
-  backup_config_files
+end
 
-  if @options[:logical_backup]
-    @databases.each do |database|
-      online_backup(database)
-    end
-  end
+puts "Starting backup: #{Time.now}"
+create_directories(@dir.dup)
+FileUtils.cd @dir
```

Comment:
This format I find hard to follow what operations need to be run inside of `@dir`. I would find this a bit easier to read and understand if the relevant bits were wrapped in `Dir.chdir(@dir) do end` blocks.